src/bmson.rs

```diff
@@ -138,8 +145,8 @@ pub struct BmsonInfo {
 
 /// Default mode hint, beatmania 7 keys.
 #[must_use]
-pub fn default_mode_hint() -> String {
-    "beat-7k".into()
+pub fn default_mode_hint() -> Cow<'static, str> {
+    Cow::Borrowed("beat-7k")
```

Comment:
```suggestion
pub fn default_mode_hint() -> &'static str {
    "beat-7k"
```
I think libraries should provide the most widest type for extendability and should not be weaken as a `Cow`.